validation/td-json-schema-validation.json

```diff
@@ -786,6 +786,42 @@
           }
         ]
       },
+      "genericSecurityScheme": {
+        "type": "object",
+        "properties": {
+          "@type": {
+            "$ref": "#/definitions/type_declaration"
+          },
+          "description": {
+            "$ref": "#/definitions/description"
+          },
+          "descriptions": {
+            "$ref": "#/definitions/descriptions"
+          },
+          "proxy": {
+            "$ref": "#/definitions/anyUri"
+          },
+          "scheme": {
+            "type": "string",
+            "not": {
+              "enum": [
+                "nosec",
+                "auto",
+                "apikey",
+                "combo",
+                "basic",
+                "digest",
+                "bearer",
+                "psk",
+                "oauth2"
+              ]
+            }
+          }
+        },
+        "required": [
```

Comment:
How about making `@type` required. Otherwise people can put anything in scheme with no way for others to understand it.

Reply:
Sure, that would be a possibility :) Then I guess we would need to adjust [Example 49](https://w3c.github.io/wot-thing-description/#example-49).